packages/file-asset-apis/src/s3/createS3Client.ts

```diff
@@ -0,0 +1,60 @@
+import fs from 'fs-extra';
+import { Agent } from 'https';
+import { S3Client as BaseClient } from '@aws-sdk/client-s3';
+import { NodeHttpHandler } from '@aws-sdk/node-http-handler';
+import type { S3ClientConfig as baseConfig } from '@aws-sdk/client-s3';
+import { debugLogger, has } from '@terascope/utils';
+import type { S3Client } from './client-types';
+
+export interface S3ClientConfig extends baseConfig {
+    sslEnabled?: boolean,
+    certLocation?: string,
+    httpOptions?: object,
+    secretAccessKey?: string,
+    accessKeyId?: string
+}
+
+export async function createS3Client(
+    config: S3ClientConfig,
+    logger = debugLogger('elasticsearch-client')
```

Comment:
```suggestion
    logger = debugLogger('s3-client')
```

I don't think you mean `elasticsearch-client` here do you?

Reply:
fixed